### PR TITLE
Rk/perturbation

### PR DESCRIPTION
**Pull request by rkukulski**

### CODE REVIEW

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 1]
```diff
@@ -2,142 +2,158 @@
...
 
-Let $U \in \mathrm{U}(\mathbb{C}^d)$ be a unitary matrix with spectral
-decomposition  Assume that the eigenvalue $\lambda \in \lambda(U)$ is
-such that $r(\lambda) = k$. Let us define a matrix $V(t)$ given by  Let
-$\lambda(t):=\lambda(UV(t))$ and let every $\_j(t)  (t)$corresponds to
-eigenvector$ $. Assume that $\lambda_{1}(t), \ldots, \lambda_{k}(t)$ are
-such eigenvalues that $\lambda_{j}(t) \to \lambda$, as $t \to 0$. Then:
+Let us consider two unitary matrices - matrix $U \in \mathrm{U}(\mathbb{C}^d)$
+and its perturbation $V \in \mathrm{U}(\mathbb{C}^d)$, such that $\\\|U-V\\\| \le \epsilon$, for some $\epsilon > 0$.
+To connect those unitary matrices, we define geodesic curve $U(t) \in \mathrm{U}(\mathbb{C}^d)$
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: It should be "a geodesic curve"

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 2]
```diff
@@ -2,142 +2,158 @@
...
+for any $t \in \[0,1\]$ with boundary conditions $U(0) \coloneqq U$ and $U(1) \coloneqq V$. 
+This smooth path is explicitly given as<sup>{% cite antezana2014optimal %}</sup>
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: Why the `<sup>`? We want citations to be inline with the rest of the text

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 3]
```diff
@@ -2,142 +2,158 @@
...
+This smooth path is explicitly given as<sup>{% cite antezana2014optimal %}</sup> 
+
+\begin{equation}
+t \mapsto U \exp \left( t \text{Log} \left( U^\dagger V \right) \right). 
+\end{equation}
 
-1\. If $*{ S*^U} \_{i=1}^d\\ p\_i ||^2=0$, then $\lambda$ is an
-eigenvalue of $UV(t)$;
+We will study how the numerical range $W(U(t))$ changes depending on $t$. As shown in 
+{% cite kukulski2020 %} we may focus on equivalent, but simplified problem of determining the behavior
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md:21]: Here the citation is on the normal level. Please change all citations to this style

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 4]
```diff
@@ -2,142 +2,158 @@
...
+of the numerical range of $U(t) = U \exp \left( \ii t D_+ \right)$, where $D_+$ is diagonal, positive, 
+semi-definite matrix, such that $\tr (D_+) = 1$.
 
-2\. If $\|\\{p_i: p_i\>0\\}\|=l\<k$, then $\lambda$ is an eigenvalue of
-$UV(t)$ and $r(\lambda) \ge k-l$;
+## Theorem<sup>{% cite kukulski2020 %}</sup>
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: Again - citation in the upper index.

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 5]
```diff
@@ -2,142 +2,158 @@
...
+\begin{equation}
+S_{\lambda}^{U} = \left\\{ \ket{x} \in \mathbb{C}^d: (\lambda \1_d - U) \ket{x} = 0, \,\, \\\| x \\\|_2 = 1 \right\\}.
+\end{equation}
+
+Let $U \in \mathrm{U}(\mathbb{C}^d)$ be a unitary matrix with spectral
+decomposition 
 
-4\. If $k=1$, then  for small $t  0$;
+\begin{aligned}
+    U = \sum_{j=1}^d \lambda_j \proj{\lambda_j}
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: Missing dot at the end of sentence

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 6]
```diff
@@ -2,142 +2,158 @@
...
+\end{aligned}
 
-5\. Let $Q:=I_{U,\lambda}^\dagger D_+ I_{U,\lambda}$ and $\lambda_1(Q)
-\le \lambda_2(Q) \le \ldots \le \lambda_k(Q)$. Then we have  for small
-$t \geq 0$ and eigenvector $\ket{ x_j}$ corresponding to $\lambda_j \in
-\lambda(U)$ is given by  where $ S^{Q}\_{\_j(Q)}$;
+Assume that the eigenvalue $\lambda \in \lambda(U)$ is such that $r(\lambda) = k$. Let us define a matrix
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md:42]: Why $\lambda(U)$ and not $\lambda \in \{ \lambda_1, \ldots, \lambda_d\}$? If we insist on $\lambda(U)$, then is it defined as a set or a multiset?

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 7]
```diff
@@ -2,142 +2,158 @@
...
+eigenvector $\ket{x_j(t)}$. Assume that $\lambda_{1}(t), \ldots, \lambda_{k}(t)$ are
+such eigenvalues that $\lambda_{j}(t) \to \lambda$, as $t \to 0$. Then:
 
-Intuitively speaking, this theorem gives us equations which one can use
-to predict behaviour of $W(UV(t))$. Observe the postulate $(6)$ fully
-determines the movement of the spectrum. However, this is a theoretical
-statement and in practice determining the function $t $ is a numerically
-complex task. The postulates $(1)-(5)$ play a key role in numerical
-calculations of $W(UV(t))$. The most important fact comes from $(3)$
+1. If $ \min\\{ \sum_{i=1}^d p_i |\braket{i}{x}|^2: \ket{x} \in S^{U}_{\lambda} \\}=0$, then $\lambda$ is an
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md:54]: What does $p_i$ stand for?

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 8]
```diff
@@ -2,142 +2,158 @@
...
+eigenvalue of $UV(t)$.
+
+2. If $\| \{p_i: p_i >0 \}\|=l\<k$, then $\lambda$ is an eigenvalue of
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: There something wrong with the expression here.

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 9]
```diff
@@ -2,142 +2,158 @@
...
+\end{equation}
+
+Moreover,
+
+\begin{equation}
+\sum_{j=1}^{d} \left| \frac{\partial}{\partial t} \lambda_j(t) \right| = 1.
+\end{equation}
+
+Intuitively speaking, this theorem gives us hints, which one can use
+to predict a behaviour of $W(UV(t))$. Observe that, the postulate $6.$ fully
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: The behaviour

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 10]
```diff
@@ -2,142 +2,158 @@
...
+determines the movenment of the spectrum. However, this is a theoretical
+statement and in practice, determining the function $t \mapsto \ket{x_j(t)}$ is a numerically
+complex task. The postulates from $1.$ to $5.$ play a key role in numerical
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md:96]: I wouldn't call these points postulates.

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 11]
```diff
@@ -2,142 +2,158 @@
...
-expressions are valid only in the neighbourhood of $t=0$. Moreover,
-sometimes we are able to precisely specify this velocities. This happens
-in the cases presented in $(1),(2)$. Whenever the calculated velocity is
-zero we know for sure that this eigenvalue will stay in the initial
-position. According to the postulate $(2)$ the same happens when the
-multiplicity of the eigenvalue is greater than the number of positive
-elements of vector $p$.
+expressions are valid only in the neighbourhood of $t=0$. 
 
 ## Illustration of above theorem
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: The above

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md [hunk 12]
```diff
@@ -2,142 +2,158 @@
...
 
-Matrix $U$ with eigenvalues $(1,e^{/6}, , , , -1, -1)$.
-![]({{ "/assets/numerical-range/examples/special_7_2.gif" | relative_url }}) We act on two subspaces
-with probability $p_1 = 1/3$ and $p\_2 = 2/3$. In this case the
-eigenvalue$$ is three fold degenerated, so it stays in the initial
-position.
+We act on two subspaces with probability $p_1 = 1/3$ and $p\_2 = 2/3$. In this case the
+eigenvalue $\ii$ is three fold degenerated, so it stays in the initial position.
 
 More examples you can find in {% cite kukulski2020 %}.
```
Comment (lpawela) [inline wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md]: More examples can be found in...

**Review by lpawela — CHANGES_REQUESTED** [review]

File: wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md (hunk 10 above)
Comment (rkukulski) [reply wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md:96]: We are using the  terminology from https://doi.org/10.1007/978-3-030-50433-5_48

**Review by lpawela — APPROVED** [review]

**Pull request merged**